paimon-core/src/main/java/org/apache/paimon/branch/TableBranch.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.branch;
+
+import org.apache.paimon.Snapshot;
+
+/** {@link TableBranch} has branch relevant information for table. */
+public class TableBranch {
+    private final String branchName;
+    private final String tagName;
+    private final Snapshot snapshot;
```

Comment:
I think branch should be a ref for a series snapshots, because we could write to a branch and generate new snapshots in the specific branch, what's this snapshot property here represents for?

Reply:
Rename the fields `tagName` and `snapshot`